clang/test/OpenMP/dispatch_unsupported.c

```diff
@@ -0,0 +1,7 @@
+// RUN: %clang_cc1 -emit-llvm -fopenmp -disable-llvm-passes %s -verify=expected
+
+// expected-error@+2 {{cannot compile this OpenMP dispatch directive yet}} 
+void a(){
+    #pragma omp dispatch
+    a();
+}
```

Comment:
Leave an empty line at the end of the file

Reply:
Added, thanks